tools/app/package.json

```diff
@@ -9,7 +9,10 @@
     "lint": "next lint",
     "clean": "rm -rf node_modules && rm -f package-lock.json && npm cache clean --force",
     "test": "jest",
-    "test:watch": "jest --watch"
+    "test:watch": "jest --watch",
```

Comment:
Jest could have its own command and maybe we could run e2e in headless mode when using the main `npm test` command. Then it would be also ran in the pipeline

Reply:
Yep, good idea!

Reply:
Implemented in [70349af](https://github.com/CyberismoCom/cyberismo/pull/117/commits/70349af39ab127c0a4d922f08d67d4e0838d4789)